### PR TITLE
On-premise notifications

### PR DESCRIPTION
**Pull request by jsani-radar**
https://linear.app/radarlabs/issue/FENCE-871/top-level-notifications-page

### CODE REVIEW

File: docs/notifications.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Notifications
+---
+
+import Alert from "../src/components/Alert";
+
+<Alert alertType="info">
+  Notifications are only available on {` `}
```
Comment (brettguenther) [inline docs/notifications.mdx:8]: can we include a sdk version requirement?

File: docs/notifications.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Radar only controls the registration of notifications on the device. Once that happens, surfacing notifications is subject to the [system limits and heuristics](https://developer.apple.com/documentation/usernotifications/unlocationnotificationtrigger#overview) that iOS enforces.
+
+<Alert alertType="info">
+  You need to be outside a geofence when it is registered on the device for the notification to fire upon entry.
+</Alert>
+
+## Analytics
+
+With Radar [Conversions](/api#log-a-conversion), you can log a metric whenever a user interacts with a notification.
```
Comment (brettguenther) [inline docs/notifications.mdx:47]: a metric > a event

File: docs/notifications.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Radar's on-premise notifications make use of location notification triggers on iOS. These triggers work with only foreground or <b>"when in use"</b> permissions. This feature is privacy-friendly and won't collect any additional location data. Notifications are only displayed after the user opens the app for the first time with the [Radar SDK](/documentation/sdk) installed.
+
+No additional lines of code are necessary to power on-premise notifications. They function completely under the hood after setup. The same calls to `Radar.trackOnce()` and `Radar.startTracking()` will return nearby geofences with notifications, which will then be registered on the device.
+
+Radar only controls the registration of notifications on the device. Once that happens, surfacing notifications is subject to the [system limits and heuristics](https://developer.apple.com/documentation/usernotifications/unlocationnotificationtrigger#overview) that iOS enforces.
+
+<Alert alertType="info">
+  You need to be outside a geofence when it is registered on the device for the notification to fire upon entry.
+</Alert>
+
```
Comment (brettguenther) [inline docs/notifications.mdx:44]: include details on frequency of delivery?

File: docs/notifications.mdx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+From there, add values for the following `metadata` keys to the geofences that will trigger notifications.
+
+###### Metadata keys
+- **`radar:notificationText`** (string, required): The body text of the notification.
+- **`radar:notificationTitle`** (string, optional): The title of the notification, displayed in bold.
+
+![PumpToStoreMetadata](/img/notifications/pump_to_store_metadata.png)
+
+Finally, [integrate the SDK](/sdk/ios) and call `Radar.trackOnce()` or `Radar.startTracking()`, depending on your use case. Make sure that the `syncGeofences` [tracking option](/sdk/tracking#ios) is enabled. Radar will display a notification when the device is detected inside a geofence that has the above `metadata` keys.
```
Comment (lmeier) [inline docs/notifications.mdx:31]: Think we should mention that indeed `syncGeofences: true` in `efficient` and `responsive`, and also maybe include a code snippet showing how to set the tracking options (or link to the call).

**Review by lmeier — APPROVED** [review]

**Review by lmeier — APPROVED** [review]

File: docs/notifications.mdx (hunk 4 above)
Comment (jsani-radar) [reply docs/notifications.mdx:31]: think the link out to the tracking options docs takes care of this as the presets table shows that `syncGeofences = true` for `efficient` and `responsive`

**Pull request merged**